tests/unit/contrib/django/test_utils.py

```diff
@@ -27,3 +29,18 @@ def test_get_settings_default():
 )
 def test_package_is_installed(package_name, expected):
     assert utils.package_is_installed(package_name) is expected
+
+
+@pytest.mark.parametrize(
+    "version, version_wanted,expected",
+    [
+        ("3.1.3", 3, True),
+        ("2.1.3", 3, False),
+        ("pytest", 3, False),
+        (None, 3, False),
+    ],
+)
+def test_package_is_version(version, version_wanted, expected):
+    with patch("importlib.metadata.version") as mock:
+        mock.return_value = version
+        assert utils.package_is_version("abcd", version_wanted) is expected
```

Comment:
```suggestion
def test_package_is_version(version, version_wanted, expected, mocker):
    mocker.patch("importlib.metadata.version", return_value=version)
    
    assert utils.package_is_version("abcd", version_wanted) is expected
```

Reply:
(Also trying to decide if it's worth calling `mock.assert_called_with("abcd")` but not sure. Seems a bit too much, let's leave it as-is)